### PR TITLE
Bugfix: changed cache suffix and style fix zf toolbar

### PR DESCRIPTION
**Pull request by thomasvargiu**
### Changes ###

- Dot (.) in cache suffix is not enabled in default filesystem cache configuration. Changed with dash (-).
- ZF toolbar dependencies list style fixes.
- Dependencies list view: outdated dependencies first.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "zendframework/zend-developer-tools":   "dev-master",
         "squizlabs/php_codesniffer":            "~1",
         "zendframework/zend-view":              "~2.3",
-        "zendframework/zend-cache":             "~2.3"
+        "zendframework/zend-cache":             "~2.3",
+        "phpunit/phpunit": "~4.2.0"
```
Comment (Ocramius) [inline composer.json]: This should be in `require-dev`

File: src/VersionEyeModule/Collector/DependencyStatusCollector.php [hunk 2]
```diff
@@ -108,11 +108,20 @@ public function collect(MvcEvent $mvcEvent)
     }
 
     /**
+     * @param bool $outdatedFirst
      * @return array|null
      */
-    public function getCollectedDependencyStatuses()
+    public function getCollectedDependencyStatuses($outdatedFirst = false)
```
Comment (Ocramius) [inline src/VersionEyeModule/Collector/DependencyStatusCollector.php]: This seems to be fixing a different issue than what is done in `composer.json`

File: composer.json (hunk 1 above)
Comment (thomasvargiu) [inline composer.json]: It's in ```require-dev```

File: src/VersionEyeModule/Collector/DependencyStatusCollector.php (hunk 2 above)
Comment (thomasvargiu) [inline src/VersionEyeModule/Collector/DependencyStatusCollector.php]: Should I separate commits?

**Comment by thomasvargiu** [comment]
any news?

File: composer.json (hunk 1 above)
Comment (Ocramius) [inline composer.json]: Nvm, totally misread

File: src/VersionEyeModule/Collector/DependencyStatusCollector.php (hunk 2 above)
Comment (Ocramius) [inline src/VersionEyeModule/Collector/DependencyStatusCollector.php]: Well, no, but is this API needed?

**Comment by Ocramius** [comment]
Sorry, totally missed this. Please see comments and ping me again :X

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by thomasvargiu** [comment]
@Ocramius 
Ok, I removed ```$outdatedFirst``` parameter. You're right, it was not necessary.

**Comment by Ocramius** [comment]
:+1:

**Pull request merged**